_studio/shared/umc/codec/h265_dec/src/umc_h265_heap.cpp

```diff
@@ -343,9 +343,6 @@ uint8_t * CumulativeArraysAllocation(int n, int align, ...)
     int cumulativeSize = 0;
     for (int i = 0; i < n; i++)
     {
-        void * ptr = va_arg(args, void *);
```

Comment:
You can't remove this va_arg call since va_arg reads an argument and advances to the next one. So, you will read not a currSize, but pointer instead.